bundles/af-core/src/test/resources/form/numberinput/test-content.json

```diff
@@ -36,6 +36,7 @@
     "fieldType": "number-input",
     "description": "Enter your phone number.",
     "name": "Phone Number",
+    "jcr:title" : "Phone Number",
```

Comment:
Why have you added this ?

Reply:
As discussed in scrum we have added jcr:title  in order to pass the tests without modifying any values in existing collateral.